bundles/org.eclipse.swt/Eclipse SWT Tests/win32/org/eclipse/swt/widgets/ControlWin32Tests.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Yatta Solutions
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Yatta Solutions - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.widgets;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Font;
+import org.eclipse.swt.graphics.FontData;
+import org.eclipse.swt.internal.DPITestUtil;
+import org.eclipse.swt.internal.DPIUtil;
+import org.eclipse.swt.internal.Win32AutoscaleTestBase;
+import org.junit.Test;
+
+/**
+ * Automated Tests for class org.eclipse.swt.widgets.Control
+ * for Windows specific behavior
+ *
+ * @see org.eclipse.swt.widgets.Control
+ */
+public class ControlWin32Tests extends Win32AutoscaleTestBase {
+
+	@Test
+	public void scaleFontCorrectlyInAutoScaleSzenario() {
```

Comment:
Please name the test methods `test...`, that makes it easier to spot tests in the call hierarchy.